### PR TITLE
[EDIT] CSS Image and Link

### PR DESCRIPTION
**Pull request by Kadif-Ravik**
### Description
Uploaded image for CSS examples of rgba.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/css/concepts/colors/terms/rgba/rgba.md [hunk 1]
```diff
@@ -51,3 +51,5 @@ h2 {
   color: rgba(0, 0, 0, 25%);
 }
 ```
+
```
Comment (ishg-153) [inline content/css/concepts/colors/terms/rgba/rgba.md:54]: Add in a sentence here, that connects the above code blocks to the output. You can explain what is happening in the code that results in the output image you added.

File: content/css/concepts/colors/terms/rgba/rgba.md [hunk 2]
```diff
@@ -51,3 +51,5 @@ h2 {
...
+![Examples 1 and 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-colors-rgba-example.png)
```
Comment (ishg-153) [inline content/css/concepts/colors/terms/rgba/rgba.md:55]:
```suggestion
![CSS color rgba example](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-colors-rgba-example.png)
```

The text inside the square brackets acts as alt text. So it should be a phrase about the output image you added, like this:

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Kadif-Ravik the image looks good. There are a couple of additional things you need to add. I have suggested those. 
Once you are done making the changes, tag me.

File: content/css/concepts/colors/terms/rgba/rgba.md (hunk 2 above)
Comment (Kadif-Ravik) [reply content/css/concepts/colors/terms/rgba/rgba.md:55]: @ishg-153 will do.

**Comment by Kadif-Ravik** [comment]
@ishg-153 Here you go.

**Review by ishg-153 — APPROVED** [review]
@Kadif-Ravik I deleted the extra image file since it isn't being used anymore.
This looks good for a second review.

**Comment by Kadif-Ravik** [comment]
@ishg-153 Thanks.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Kadif-Ravik The image doesn't match the two examples. The h1 string should be 50% opacity, but the image looks like 100%.

**Review by caupolicandiaz — APPROVED** [review]
@Kadif-Ravik LGTM

**Pull request merged**